docker-compose.yml

```
@@ -161,6 +168,7 @@ services:
       KC_DB_PASSWORD: pass
       KC_DB_USERNAME: people
       KC_DB_SCHEMA: public
+      PROXY_ADDRESS_FORWARDING: true
```

The job `e2e` doesn't seem happy with that. Considere maybe a string ?

---

Yeah I wanted to discuss it, I followed your initial implementation. What type of string? 
Btw, e2e tests need to be fixed locally !

---

Happy to dig it together !

---

https://docs.docker.com/compose/compose-file/compose-file-v3/#environment
So I guess:
```suggestion
      PROXY_ADDRESS_FORWARDING: 'true'
```

Let's see what the job e2e says, it will be easier to debug in local after that.